### ISSUE TITLE
Bootstrap template pack and django-crispy-forms in doc's cookbook

### ISSUE THREAD

**Issue by qcaron** [zsiciarz/django-envelope#32]
Hi there,

I am using django-envelope with bootstrap3 as template pack. I was trying to layout the contact form horizontally so I decided to create a custom view which uses a custom form.

Here is my forms.py:

```python
from envelope.forms import ContactForm
from crispy_forms.helper import FormHelper

class MyContactForm(ContactForm):
    subject_intro = "yop"
    template_name = "plaintext_email.txt"
    html_template_name = "contact_email.html"

    def __init__(self, *args, **kwargs):
        super(MyContactForm, self).__init__(*args, **kwargs)
        self.helper = FormHelper()
        self.helper.form_class = 'form-horizontal' # just a reminder because explicitly used in my template
        self.helper.label_class = 'col-lg-2'
        self.helper.field_class = 'col-lg-10'
```

So as you can see I am using django-cripy-forms to customize the default layout. Here is my form display in templates/envelope/contact.html:

```html
<form action="{% url 'envelope-contact' %}" method="post" class="form-horizontal">
    {% csrf_token %}
    {% antispam_fields %}
    {% crispy form %} <!-- django-crispy-forms tag to adapt display according to MyContactForm -->
    <input type="submit" class="btn btn-default" value="Submit">
</form> 
```

I scratched my head for like 30 mins because I did not know about django-crispy-forms yet. The difference here in my template is that I do not use  {% form | crispy %} as stated in the doc's cookbook but {% crispy form %}.

Is this really what I should be doing? If yes, I believe it worths a small documentation update to avoid others to spend time understanding what is wrong :)

Cheers,
Quentin

**Comment by zsiciarz** [comment]
Hi Quentin!
Thanks for trying out `django-envelope`. From my experience with `django-floppy-forms`, the `|crispy` filter is the simplest way to render the form. However - if you're using a custom layout helper (like you do above), the `{% crispy ... %}` template tag is the way to go ([source](http://django-crispy-forms.readthedocs.org/en/latest/filters.html)). Therefore you're right - it makes sense to go with the more general template tag instead of filter. I'll comment on your PR in more detail.

**Comment by zsiciarz** [comment]
Merged in #33.
